frontend/yarn.lock

```diff
@@ -1123,7 +1123,7 @@
 
 "@babel/runtime-corejs3@^7.12.1":
   version "7.16.3"
-  resolved "https://registry.yarnpkg.com/@babel/runtime-corejs3/-/runtime-corejs3-7.16.3.tgz#1e25de4fa994c57c18e5fdda6cc810dac70f5590"
```

Comment:
Do you know why these hashes show up here? Might cause some conflicts with other branches